rules/otp.bzl

```diff
@@ -207,3 +208,13 @@ otp_image = rule(
         ),
     },
 )
+
+# This is a set of overlays to generate a generic, standard OTP image.
+# Additional overlays can be applied on top to further customize the OTP.
+STD_OTP_OVERLAYS = [
```

Comment:
Note: this would really make more sense in `//hw/ip/otp_ctrl/data/BUILD`, but in order to use the variable in other files, it must be defined in a .bzl file. This seemed like the next most logical place to put it.